apps/platform-app/prisma/migrations/20241203154903_add_cv_url_column_to_profile_model/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Profile" ADD COLUMN     "cvUrl" TEXT;
```

Comment:
**suggestion**: Consider adding a comment explaining the purpose of adding the 'cvUrl' column to the 'Profile' table for better context.